package.json

```diff
@@ -86,7 +86,7 @@
       "modes-of-issuance": "1.0",
       "nature-of-content-terms": "1.0",
       "search": "0.7 1.0 1.3",
-      "source-storage-records": "3.0",
+      "source-storage-records": "3.3",
```

Comment:
This is a breaking change; you are dropping support for everything less than `3.3`. That's fine, the major version has already been bumped, but please call out this breaking change in the CHANGELOG.